spec/features/npq_separation/admin/finance/statement_spec.rb

```diff
@@ -25,6 +25,12 @@
     sign_in_as(create(:admin))
   end
 
+  scenario "shows side navigation with current statement highlighted" do
+    visit(npq_separation_admin_finance_statement_path(statement))
+
+    expect(page).to have_content("Finance statement")
```

Comment:
```suggestion
    within "#side-navigation" do |side_navigation|
      expect(side_navigation).to have_content("Finance statement")
      expect(side_navigation).to have_content("#{statement.lead_provider.name}, #{Date::MONTHNAMES[statement.month]} #{statement.year}")
    end
```
(you'll have to rebase on `main` to get the `#side-navigation` to work)